webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -13,15 +13,26 @@ export function notify({message, type, link, sticky = isSticky(type)}) {
       }));
 }
 
+export function clear() {
+  store.dispatch(ToastActions.clearToasts());
+}
+
 function importFlashMessagesFromRails() {
-  $('#notifications').data().flash.forEach(([type, message]) => {
+  const notifications = $('#notifications');
+
+  if (notifications.length === 0) {return;}
+
+  notifications.data().flash.forEach(([type, message]) => {
```

Comment:
This seems to fail when there is no flash data, which occurs in some of the tests.